### PR TITLE
Add save-to-cubox extension

### PR DESCRIPTION
**Pull request by xyxc0673**
## Description

Save url or memo to your cubox service

## Screencast

CleanShot 2023-11-25 at 20 28 19@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/save-to-cubox/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      "name": "save-to-cubox",
+      "title": "Save to Cubox",
+      "description": "Save url or memo to your cubox",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "api",
+      "title": "Cubox API",
```
[the remark this answers is not in the document]
Comment (xyxc0673) [reply extensions/save-to-cubox/package.json:23]: Cubox is a web service and provides API like https://cubox.pro/c/api/save/ttttttt, so I think use Cubox  API as the field title will be more accurate.

**Comment by xyxc0673** [comment]
Hi

I have used `useForm` instead and add a metadata image.
Also handle error gracefully.

File: extensions/save-to-cubox/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (pernielsentikaer) [reply extensions/save-to-cubox/package.json:23]: Could we atleast add a placeholder so the user can see the format?

**Comment by pernielsentikaer** [comment]
Looks really good, the last placeholder and we're about to be ready 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**